### PR TITLE
Feature/2 optimisation algorithm

### PR DESCRIPTION
**Pull request by GuillemBarroso**
This is the first PR sovling issue #2 !!

### CODE REVIEW

File: script.py [hunk 1]
```diff
@@ -1,2 +1,9 @@
 tst = 1
 asdasdsadasd = 1
+test 2 is about to be pushed to origin
```
Comment (Aiguasol) [inline script.py]: This is the second inline comment

[the remark this answers is not in the document]
Comment (GuillemBarroso) [reply script.py]: Test

Comment (GuillemBarroso) [reply script.py]: TEst2

Comment (Aiguasol) [reply script.py]: you should do this

**Pull request merged**